PWGLF/DataModel/LFStrangenessTables.h

```diff
@@ -180,6 +195,13 @@ DECLARE_SOA_TABLE(MotherMCParts, "AOD", "MOTHERMCPART", //! mother MC informatio
 
 using MotherMCPart = MotherMCParts::iterator;
 
+DECLARE_SOA_TABLE(StoredMotherMCParts, "AOD1", "MOTHERMCPART", //! mother MC information, abbreviated name due to size limit
```

Comment:
Idem

Reply:
Same reply as above. These are generally needed when we want to make derived data. Currently on the first PR we haven't implemented the derived data part itself yet (this we will do next). But we wanted to make the changes needed already to the LF code so that we hopefully wouldn't need to bother you in future PRs.

Reply:
idem